unity-client/Assets/ABConverter/Client.cs

```diff
@@ -196,6 +196,60 @@ public static Core.State ConvertScenesToAssetBundles(List<string> sceneCidsList,
             return core.state;
         }
 
+        /// <summary>
+        /// This will start the asset bundle conversion for a single asset
+        /// </summary>
+        /// <param name="assetHash">The asset's content server hash</param>
+        /// <param name="assetFilename">The asset's content server file name</param>
+        /// <param name="sceneCid">The asset scene ID</param>
+        /// <param name="settings">Any conversion settings object, if its null, a new one will be created</param>
+        /// <returns>A state context object useful for tracking the conversion progress</returns>
+        public static Core.State ConvertAssetToAssetBundle(string assetHash, string assetFilename, string sceneCid, Settings settings = null)
+        {
+            if (string.IsNullOrEmpty(assetHash))
+            {
+                log.Error("Missing asset hash for ConvertAssetToAssetBundle()");
+                return new Core.State() {lastErrorCode = Core.ErrorCodes.UNDEFINED};
+            }
+
+            if (string.IsNullOrEmpty(assetFilename))
+            {
+                log.Error("Missing asset file name for ConvertAssetToAssetBundle()");
+                return new Core.State() {lastErrorCode = Core.ErrorCodes.UNDEFINED};
+            }
+
+            log.Info($"Building {assetHash} asset...");
+
+            EnsureEnvironment();
+
+            if (settings == null)
+            {
+                settings = new Settings()
+                {
+                    skipAlreadyBuiltBundles = false
+                };
+            }
+
+            var core = new ABConverter.Core(env, settings);
+
+            List<ContentServerUtils.MappingPair> rawContents = new List<ContentServerUtils.MappingPair>();
+            rawContents.Add(new ContentServerUtils.MappingPair
+            {
+                file = assetFilename,
+                hash = assetHash
+            });
+
+            // If the asset is a GLTF we add the dependencies to the rawContents to be downloaded
+            if (assetFilename.Contains(".glb") || assetFilename.Contains(".gltf") || assetFilename.Contains(".GLB") || assetFilename.Contains(".GLTF"))
```

Comment:
What do you think about reducing the number of conditions using `toLower()` in this way?

```suggestion
            if (assetFilename.toLower().Contains(".glb") || assetFilename.toLower().Contains(".gltf"))
```

Reply:
Is `assetFilename` expected to have another extension while being a GLB?
I mean, Is there any particular reason of using `Contains` instead of `EndsWith`?
ie: `assetFilename.EndsWith(".gltf", StringComparison.OrdinalIgnoreCase);`

Reply:
true that, `EndsWith()` should work as well, I'll try it out